frontend/src/metabase/selectors/settings.ts

```diff
@@ -51,6 +51,11 @@ export const getStoreUrl = (path: StorePaths = "") => {
   return `https://store.metabase.com/${path}`;
 };
```

Comment:
should we deprecate `getStoreUrl()` ? It's only used in 9 files, can we just remove it in favor of a more correct version?